From: SeongJae Park <sj@kernel.org>
Date: Mon, 12 May 2025 17:50:08 -0700
Subject: [PATCH] samples/damon/wsse: rename to have damon_sample_ prefix

DAMON sample module, wsse has its name 'wsse'.  It could conflict with
future modules, and not very easy to idnetify it by name.  Use a prefix,
"damon_sample_" for the name.

Signed-off-by: SeongJae Park <sj@kernel.org>
---
 samples/damon/wsse.c | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/samples/damon/wsse.c b/samples/damon/wsse.c
index e20238a249e7..1e8969a0a626 100644
--- a/samples/damon/wsse.c
+++ b/samples/damon/wsse.c
@@ -12,6 +12,11 @@
 #include <linux/kernel.h>
 #include <linux/module.h>
 
+#ifdef MODULE_PARAM_PREFIX
+#undef MODULE_PARAM_PREFIX
+#endif
+#define MODULE_PARAM_PREFIX "damon_sample_wsse."
+
 static int target_pid __read_mostly;
 module_param(target_pid, int, 0600);
 
-- 
2.39.5

